chain/block.go

```diff
@@ -609,7 +609,12 @@ func (b *StatelessBlock) innerVerify(ctx context.Context, vctx VerifyContext) er
 	heightKeyStr := string(heightKey)
 	timestampKeyStr := string(timestampKey)
 	feeKeyStr := string(feeKey)
-	tsv := ts.NewView(set.Of(heightKeyStr, timestampKeyStr, feeKeyStr), map[string][]byte{
+
+	keys := make(state.Keys)
+	keys.AddKeyAndPermission(heightKeyStr, state.NewPermission(state.Write))
+	keys.AddKeyAndPermission(timestampKeyStr, state.NewPermission(state.Write))
+	keys.AddKeyAndPermission(feeKeyStr, state.NewPermission(state.Write))
```

Comment:
nit: why not call this `state.Permission`? I don't think of us as creating a "new" permission, rather referencing existing ones.